src/crun.c

```diff
@@ -203,6 +203,12 @@ print_version (FILE *stream, struct argp_state *state arg_unused)
 #endif
 #ifdef HAVE_CRIU
   fprintf (stream, "+CRIU ");
+#endif
+#ifdef HAVE_MOVE_MOUNT
+  fprintf (stream, "+MOVE_MOUNT ");
+#endif
+#ifdef HAVE_OPEN_TREE
+  fprintf (stream, "+OPEN_TREE ");
 #endif
   fprintf (stream, "+YAJL\n");
```

Comment:
I see you are using this information for the tests.

Is there any other way we could check this out without embedding this information in the build flags?